lib/wallaby/experimental/chrome.ex

```diff
@@ -0,0 +1,161 @@
+defmodule Wallaby.Experimental.Chrome do
+  @behaviour Wallaby.Driver
+
+  @pool_name Wallaby.ChromedriverPool
+
+  alias Wallaby.Session
+  alias Wallaby.Experimental.Chrome.Webdriver
+  # alias Wallaby.Phantom.Driver
+  alias Wallaby.Experimental.Selenium.WebdriverClient
+
+  def child_spec(), do: :poolboy.child_spec(@pool_name, poolboy_config(), [])
+
+
+  def start_session(opts \\ []) do
+    chromedriver = :poolboy.checkout(@pool_name, true, :infinity)
+    start_session(chromedriver, opts)
+  end
+
+  def start_session(chromedriver, opts) do
+    {:ok, base_url} = Wallaby.Experimental.Chrome.Chromedriver.base_url(chromedriver)
+    capabilities = Keyword.get(opts, :capabilities, %{})
+    create_session_fn = Keyword.get(opts, :create_session_fn,
+                                    &Webdriver.create_session/2)
+
+    capabilities = Map.merge(default_capabilities(), capabilities)
+
+    with {:ok, response} <- create_session_fn.(base_url, capabilities) do
+      id = response["sessionId"]
+
+      session = %Wallaby.Session{
+        session_url: base_url <> "session/#{id}",
+        url: base_url <> "session/#{id}",
+        id: id,
+        driver: __MODULE__,
+        server: chromedriver,
+      }
```

Comment:
overall the fucntion is a bit long for my taste (yeah yeah I know...) maybe we could extract the driver building into a separate function? I think that'd get it down considerably and be more of a nicer higher level overview?